src/main/webapp/js/common/react/tests/Navbar.test.js

```diff
@@ -2,59 +2,86 @@ import React from 'react';
 import Navbar from '../components/Navbar.js';
 import '@testing-library/jest-dom/extend-expect';
 import {render, screen} from '@testing-library/react';
-import fetchMock from 'fetch-mock-jest';
-import {makeRelativeUrlAbsolute} from '../../../fetch_handler.js';
 
 const navbarUrls = [
   {
     href: '/',
     name: 'Home',
+    requiresAuth: false,
   },
   {
     href: '/projects.html',
     name: 'Projects',
+    requiresAuth: false,
+  },
+  {
+    href: '/dashboard.html',
+    name: 'Dashboard',
+    requiresAuth: true,
   },
```

Comment:
Is there a reason you don't include the mentor search page in this test?

Reply:
Not really, no. Joe commented on this as well in the MVP branch, so I'll be releasing an additional PR that revises this to use the data directly from `display_navbar.js`